package.json

```diff
@@ -39,6 +39,6 @@
     ]
   },
   "engines": {
-    "node": ">=16.14"
+    "node": ">=18.13"
```

Comment:
Node 16 is EOL now: https://nodejs.dev/en/about/releases/

Regarding package-lock.json -> yarn.lock, our instructions say to use yarn instead of npm, and yarn rightly complains: 

```sh
$ yarn
yarn install v1.22.19
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/5] Validating package.json...
```